nude.py

```diff
@@ -19,15 +18,8 @@ def is_nude(path_or_io):
 
 class Nude(object):
 
-    Skin = namedtuple("Skin", "id skin region x y checked")
-
     def __init__(self, path_or_io):
-        if isinstance(path_or_io, Image.Image):
```

Comment:
The removal of this code may be a breaking change, unless `Image.open()` can accept an `Image.Image` instance too (perhaps it just returns it back).